app/js/common.js

```diff
@@ -0,0 +1,38 @@
+function createCalendar(id, year, month) {
```

Comment:
you copy this solution from - https://next.plnkr.co/edit/rE7ujjEo00sQoTTkyl0s?p=preview&preview